packages/paths/src/__tests__/fixtures/graphqlCodeGen/nonExistingField/.redwood/schema.graphql

```diff
@@ -0,0 +1,31 @@
+scalar BigInt
```

Comment:
These seem unrelated... related to introspection/gql parsing. Should we remove the unneeded tests?